web/src/main/java/com/epam/rft/atsy/web/validation/messagekey/PasswordValidationMessageKey.java

```diff
@@ -0,0 +1,24 @@
+package com.epam.rft.atsy.web.validation.messagekey;
```

Comment:
In my opinion this enum should be moved to com.epam.rft.atsy.web.mapper package, because the only class which use this enum is PasswordValidationRuleMapper, and if I know well, it is not necessary to use this enum elsewhere, so if this enum is moved to the PasswordValidationRuleMapper's package, then this enum's visibility modifier can be "package-private" (the default one).